src/main/java/seedu/address/logic/commands/ExportCommand.java

```diff
@@ -202,11 +205,26 @@ public CsvSchema buildCsvSchema(JsonNode jsonTree) {
      */
     public void writeToCsvFile(File csvFile, JsonNode jsonTree) throws IOException {
         CsvSchema csvSchema = buildCsvSchema(jsonTree);
-
         CsvMapper csvMapper = new CsvMapper();
+
+        // The Code below removes examscores from the json tree
+        // Potential future implementation: Export Command also exports exam scores
+
+        // Create a new array node to hold the modified json nodes
+        ArrayNode modifiedJsonTree = JsonNodeFactory.instance.arrayNode();
+        // Iterate over each json node in the json tree
+        for (JsonNode jsonNode : jsonTree) {
```

Comment:
Perhaps in the future we implement the creation of the JsonTree to include exam scores